chatto/message.py

```diff
@@ -37,22 +38,48 @@
 from chatto.channel import Channel
 from chatto.stream import Stream
 
+class MessageTypes(enum.Enum):
+    ChatEndedEvent = "chatEndedEvent"
+    MessageDeletedEvent = "messageDeletedEvent" 
+    NewSponsorEvent = "newSponsorEvent"
+    SponsorOnlyModeEndedEvent = "sponsorOnlyModeEndedEvent"
+    SponsorOnlyModeStartedEvent = "sponsorOnlyModeStartedEvent"
+    MemberMilestoneChatEvent = "memberMilestoneChatEvent"
+    SuperChatEvent = "superChatEvent"
+    SuperStickerEvent = "superStickerEvent"
+    TextMessageEvent = "textMessageEvent"
+    Tombstone = "tombstone"
+    UserBannedEvent = "userBannedEvent"
```

Comment:
Enum keys should be upper-snake-case. For example, line 42 would look like:
```py
CHAT_ENDED_EVENT = "chatEndedEvent"
```
Remember to make this change elsewhere in the code too.

Reply:
on it